types/npm-packlist/index.d.ts

```diff
@@ -1,18 +1,22 @@
 // Type definitions for npm-packlist 3.0
```

Comment:
```suggestion
// Type definitions for npm-packlist 7.0
```
when bumping a version, this one should match target version
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
Oh, thanks!